src/model_theory/terms_and_formulas.lean

```diff
@@ -185,8 +191,16 @@ realize_bounded_formula M f v fin_zero_elim
 @[reducible] def realize_sentence (φ : L.sentence) : Prop :=
 realize_formula M φ pempty.elim
 
+/-- A sentence can be evaluated as true or false in a structure. -/
+@[reducible] def realize_theory (T : L.Theory) : Prop :=
+∀ φ ∈ T, realize_sentence M φ
```

Comment:
If you instead call this `Theory.realize` then I think you would be able to use dot notation as `T.realize` later.

Reply:
I'm going to try using dot notation, and also perhaps call it `T.model M`. I'm not sure how best to use dot notation to capture the normal phrasing: that `M` models `T`, or that `M` is a model of `T`.